jobs/email-reminder/email_reminder.py

```diff
@@ -138,9 +139,40 @@ async def find_and_send_ar_reminder(app: Flask, qsm: QueueService):  # pylint: d
         app.logger.error(err)
 
 
+async def send_outstanding_bcomps_ar_reminder(app: Flask, qsm: QueueService):  # pylint: disable=redefined-outer-name
+    """Find outstanding bcomps to send annual report reminder."""
+    try:
+        # get token
+        token = AccountService.get_bearer_token()
+        ar_fee = get_ar_fee(app, Business.LegalTypes.BCOMP.value, token)
+
+        app.logger.debug('Getting outstanding bcomps to send AR reminder')
+        where_clause = text(
+            'CASE WHEN last_ar_date IS NULL THEN date(founding_date) ELSE date(last_ar_date) END' +
+            " <= CURRENT_DATE - interval '1 year'")
+        businesses = db.session.query(Business).filter(
+            Business.legal_type == Business.LegalTypes.BCOMP.value,
+            where_clause
+        ).all()
+        app.logger.debug('Processing outstanding bcomps to send AR reminder')
+
+        for business in businesses:
+            ar_year = (business.last_ar_year if business.last_ar_year else business.founding_date.year) + 1
+
+            await send_email(business.id, ar_fee, str(ar_year), app, qsm)
+            app.logger.debug(f'Successfully queued ar reminder for business id {business.id} for year {ar_year}.')
+
+    except Exception as err:  # pylint: disable=broad-except, unused-variable # noqa F841;
+        app.logger.error(err)
+
+
 if __name__ == '__main__':
+    condition = sys.argv[1] if sys.argv and len(sys.argv) > 1 else None
```

Comment:
@thorwolpert I was trying to figure out how to run this. I was trying `oc create job --from=cronjob/email-reminder-dev email-reminder-outstanding-bcomps -n cc892f-dev -- python /opt/app-root/email_reminder.py outstanding-bcomps` but i received an error `cannot specify --from and command`. Can you suggest me a better way to run this in os

Reply:
If the cronjob already exists
```oc -n k3s create job --from=cronjob/my-job-to-test 8888-test-run_01```
eg
``` oc -n <my namespace> --from=cronjob/<cronjob name>  <name of the job to see it in the job history>```

Reply:
I understood this. What I want to do is to pass an argument `outstanding-bcomps`